exonum-java-binding-core/src/main/java/com/exonum/binding/crypto/CryptoUtils.java

```diff
@@ -16,23 +16,56 @@
 
 package com.exonum.binding.crypto;
 
+import com.goterl.lazycode.lazysodium.LazySodium;
+
 /**
  * Utils for crypto system.
  */
 class CryptoUtils {
 
   /**
    * Check that {@code data} byte array has specified {@code size}.
+   *
    * @throws NullPointerException if it is {@code null}
    */
   static boolean hasLength(byte[] data, int size) {
     return data.length == size;
   }
 
   /**
-   * Check that returned status is a success status.
+   * Converts hexadecimal to bytes.
+   *
+   * @param hex hexadecimal string
+   * @return bytes array
+   */
+  static byte[] hexToByteArray(String hex) {
```

Comment:
I see from the implementation it uses upper case. We use lower for HashCodes, and I'd keep that behaviour consistent across these similar classes. We may take `com.exonum.binding.test.Bytes.HEX_ENCODING` as a ready-to-use example.

Would you please add a simple test for that (for these methods or, more importantly, for public classes)?